src/booster.py

```diff
@@ -0,0 +1,84 @@
+import shutil
+import subprocess
+from pathlib import Path
+
+import typer
+import yaml
+
+app = typer.Typer()
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+OUTPUT_DIR = BASE_DIR / "output"
+DEFAULT_CONFIG = BASE_DIR / "sample_config.yml"
+
+
+def load_config(config_path: Path):
+    """Load project settings from YAML configuration file."""
+    with open(config_path, "r") as file:
+        config = yaml.safe_load(file)
+    return config
+
+
+def django_start_project(config):
+    """Create a Django project using the specified configuration."""
+    project_name = config["project_name"]
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Check this is a valid Python package name

Since `project_name` is used as the package name, it needs to be a valid. We should do some basic checks here to ensure it is:

- a-z, 0-9
- Lowercase
- underscores 
- ‏